assembl/static2/js/app/reducers/screenDimensionsReducers.js

```diff
@@ -0,0 +1,13 @@
+const singleReducer = (type, key, initialState) => {
+  return (state = initialState, action) => {
+    switch (action.type) {
+    case type:
+      if (key in action) return action[key]; // eslint-disable-line no-fallthrough
+    default:
+      return state;
+    }
+  };
+};
+
+export const screenWidth = singleReducer('UPDATE_SCREEN_DIMENSIONS', 'newWidth', 0);
```

Comment:
rename to `singleActionReducer` maybe?

Reply:
well it's more singleActionSingleKeyReducer ^^

Reply:
actionKeyDefaultReducer?

Reply:
or maybe `basicReducer` or `simpleReducer` ? I think I already saw something like that in the code. I'm not a big fan since it demands some effort to understand what it is (if you want to keep it, a comment would be welcome to explain the purpose of the function (because renaming will not be sufficient))